src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py

```diff
@@ -162,34 +171,45 @@ def flexible_server_create(cmd, client,
                                    auth_config=auth_config,
                                    cluster=cluster)
 
+    # Add Microsoft Entra Admin
+    if is_microsoft_entra_auth_enabled and admin_name is not None or admin_id is not None:
+        server_admin_client = cf_postgres_flexible_adadmin(cmd.cli_ctx, '_')
+        logger.warning("Add Microsoft Entra Admin '%s'.", admin_name)
+        _create_admin(server_admin_client, resource_group_name, server_name, admin_name, admin_id, admin_type)
+
     # Adding firewall rule
     if start_ip != -1 and end_ip != -1:
         firewall_id = create_firewall_rule(db_context, cmd, resource_group_name, server_name, start_ip, end_ip)
 
-    # Create mysql database if it does not exist
+    # Create database if it does not exist
     if (database_name is not None or (create_default_db and create_default_db.lower() == 'enabled') and create_cluster != 'ElasticCluster'):
         db_name = database_name if database_name else DEFAULT_DB_NAME
         _create_database(db_context, cmd, resource_group_name, server_name, db_name)
+    else:
+        db_name = POSTGRES_DB_NAME
 
-    user = server_result.administrator_login
+    user = server_result.administrator_login if is_password_auth_enabled else '<user>'
```

Comment:
This lines causes credscan to fail:

```
##[error]1. Credential Scanner Error CSCAN-GENERAL0030 - File: src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py:src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py. Line: 193. Column 3. 
Signature: 4437f095d08224d8fa3ba459bdf3c04eda1552b093c54217622502e91385ca39
Tool: Credential Scanner: Rule: CSCAN-GENERAL0030 (User Login Credentials). https://aka.ms/credscan
A potential secret was detected in 'flexible_server_custom_postgres.py':(CSCAN-GENERAL0030 User Login Credentials) Validate file contains secrets, remove, roll credential, and use approved store. For additional information on secret remediation see https://aka.ms/credscan.
```

Reply:
How would you suggest we update this line to avoid error in future?